ladder/members.yaml

```diff
@@ -143,3 +143,5 @@ members:
 - youssefazrak
 - ysksuzuki
 - zacharysarah
+emeritus_members:
+- danehans
```

Comment:
nit: We'll probably want this in a separate file for ease of processing - while this is called `members.yaml`, it's intended to match this team: https://github.com/orgs/cilium/teams/organization-members, and I think the intent behind the Emeritus team is that it is a different group. So far all the individual files in this repo relate to a different team, almost all using the name of the file as the name of the team.

That said we'll likely need to make some changes to the format when we start processing these files with software for automation, so it's not a big deal where it goes until we embark on that effort.